src/sv-pipeline/04_variant_resolution/scripts/split_variants.py

```diff
@@ -3,72 +3,55 @@
 import logging
 
 
-# Function to process the bed file by checking for conditions
 def process_bed_file(input_bed, n_per_split, bca=True):
     SVTYPE_FIELD = 4
     END_FIELD = 2
     START_FIELD = 1
 
-    # Dictionary to store the conditions to be checked with matching prefixes
     condition_prefixes = {
-        'gt5kb': {
-            'condition': lambda line: (line[SVTYPE_FIELD] == 'DEL' or line[SVTYPE_FIELD] == 'DUP') and (int(line[END_FIELD]) - int(line[START_FIELD]) >= 5000)},
-        'lt5kb': {
-            'condition': lambda line: (line[SVTYPE_FIELD] == 'DEL' or line[SVTYPE_FIELD] == 'DUP') and (int(line[END_FIELD]) - int(line[START_FIELD]) < 5000)},
-        'bca': {'condition': lambda line: bca and (
-                line[SVTYPE_FIELD] != 'DEL' and line[SVTYPE_FIELD] != 'DUP' and line[SVTYPE_FIELD] != 'INS')},
+        'gt5kb': {'condition': lambda line: (line[SVTYPE_FIELD] == 'DEL' or line[SVTYPE_FIELD] == 'DUP') and (int(line[END_FIELD]) - int(line[START_FIELD]) >= 5000)},
+        'lt5kb': {'condition': lambda line: (line[SVTYPE_FIELD] == 'DEL' or line[SVTYPE_FIELD] == 'DUP') and (int(line[END_FIELD]) - int(line[START_FIELD]) < 5000)},
+        'bca': {'condition': lambda line: bca and line[SVTYPE_FIELD] not in ['DEL', 'DUP', 'INS']},
         'ins': {'condition': lambda line: bca and line[SVTYPE_FIELD] == 'INS'}
     }
 
     current_lines = {prefix: [] for prefix in condition_prefixes.keys()}
     current_counts = {prefix: 0 for prefix in condition_prefixes.keys()}
     current_suffixes = {prefix: 'a' for prefix in condition_prefixes.keys()}
 
-    # Open the bed file and process
     with open(input_bed, 'r') as infile:
         for line in infile:
-            # process bed file line by line
-            line = line.strip().split('\t')
-
-            # Checks which condition and prefix the current line matches and appends it to the corresponding
-            # array and increments the counter for that array
+            line = line.strip('\n').split('\t')
+            line[4], line[5] = line[5], line[4]
+            SVTYPE_FIELD = 5
```

Comment:
Instead of reassigning SVTYPE_FIELD here, you should either set SVTYPE_FIELD to 5 at the beginning or (my preference) move the code that swaps the fields to right before you append a new line to current_lines

Reply:
This was addressed and the value was set to 5.